### DISCUSSION TITLE
Ethers-v6 support

### DISCUSSION THREAD

**Discussion by zavelevsky** [NoahZinsmeister/web3-react#785] [Q&A]
Hi, is there a plan to support ethers v6 out of the box?
Alternatively, is there any guide how to somehow adapt the different types to what's expected by ethers-v6?
Has anyone been successful upgrading their project to use ethers v6 - while using web3-react?

**Comment by faces-of-eth** [comment]
Here to +1 this.

**Comment by chadzxrex** [comment]
Here for the upgrading!  +1
